README-CONAN.md

```diff
@@ -201,12 +208,12 @@ cmd
 
 ### Profiles for Visual Studio
 
-You can build Exiv2 with Visual Studio 2017 (version 15), 2015 (version 14), 2013 (version 12), 2012 (version 11), 2010 (version 10) or 2008 (version 9).
-You create profiles in %HOMEPATH%\.conan\profiles with a text editor.  For your convenience, you'll find profiles in **<exiv2dir>/cmake/msvc\_conan\_profiles**.  There are 24 in total:
+Exiv2 can be build with any Visual Studio version with c++11 support: 2017 (version 15) or 2015 (version 14).
```

Comment:
Since we do not support non-C++11 compilers anymore, I updated this section and I also removed the provided profiles that are no longer needed.